src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/preview.html.twig

```diff
@@ -0,0 +1,154 @@
+                <strong>{{ 'Invoice details'|trans({}, 'Admin.Global') }}:</strong>
+              </p>
+
+              <p class="mb-0">{{ orderPreview.invoiceDetails.firstName }} {{ orderPreview.invoiceDetails.lastName }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.company }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.address1 }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.address2 }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.country }}</p>
+              <p class="mb-0">{{ orderPreview.invoiceDetails.phone }}</p>
+              <p>{{ orderPreview.invoiceDetails.email }}</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="offset-1 col-7">
+      <div class="table-responsive">
+        <table class="table product table-borderless border-bottom-0">
+          <thead>
+          <tr>
+            {% if orderPreview.taxIncluded %}
+              {% set taxInclusion = 'Tax included'|trans({}, 'Admin.Global') %}
+            {% else %}
+              {% set taxInclusion = 'Tax excluded'|trans({}, 'Admin.Global') %}
+            {% endif %}
+
+            <th>{{ 'Product'|trans({}, 'Admin.Global') }}</th>
+            <th>{{ 'Reference'|trans({}, 'Admin.Global') }}</th>
+            <th>{{ 'Stock location'|trans({}, 'Admin.Global') }}</th>
+            <th class="text-center">
+              {{ 'Tax'|trans({}, 'Admin.Global') }}
+            </th>
+            <th class="text-center">{{ 'Quantity'|trans({}, 'Admin.Global') }}</th>
+            <th class="text-center">
+              {{ 'Total'|trans({}, 'Admin.Global') }}
+              <small>{{ taxInclusion }}</small>
+            </th>
+          </tr>
+          </thead>
+          <tbody>
+          {% for productDetail in orderPreview.productDetails %}
+            {% if loop.index <= productsPreviewLimit %}
+              <tr>
+                <td class="p-1">{{ productDetail.name }}</td>
+                <td class="p-1">{{ productDetail.reference }}</td>
+                <td class="p-1">
+                  {% if productDetail.location is not empty %}{{ productDetail.location }}{% else %}-{% endif %}
+                </td>
+                <td class="p-1 text-center">{{ productDetail.totalTax }}</td>
+                <td class="p-1 text-center">{{ productDetail.quantity }}</td>
+                <td class="p-1 text-center">{{ productDetail.totalPrice }}</td>
+              </tr>
+            {% endif %}
+          {% endfor %}
+          </tbody>
+        </table>
+      </div>
+
+      <div class="text-right">
+        <a href="{{ path('admin_orders_view', {'orderId': orderId}) }}" class="btn btn-primary btn-sm">
+          {% if orderPreview.productDetails|length > productsPreviewLimit %}
+            {{ 'View the other %count% products'|trans({'%count%': orderPreview.productDetails|length - productsPreviewLimit}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
Do we agree that this will be a small call-to-action to list the other product(s) of the order inside the preview line? Or does it redirects to the order details page?

Reply:
It will redirect to Orders View page.

Reply:
In my opinion, it is pretty obvious that we cannot list all the products in a cart in the preview, so there is no need to emphasize it with a wording. We could suggest it with a design effect and therefore keep the number of products in the cart as in the mockup and keep one call-to-action at the end 'Order details'. I'll be at the stand-up tomorrow so we can discuss it all together. :-)

Reply:
For me, anythings works, we already changed this text following feedback from Product team, but it seems it was not enough.